src/Entity/ShopBillingDataInterface.php

```diff
@@ -13,19 +13,35 @@
 
 namespace Sylius\RefundPlugin\Entity;
 
-interface ShopBillingDataInterface
+use Sylius\Component\Resource\Model\ResourceInterface;
+
+interface ShopBillingDataInterface extends ResourceInterface
 {
-    public function id(): ?int;
```

Comment:
Deprecate this method instead of removing it now

Reply:
This might come in handy 🙂  : https://github.com/Sylius/RefundPlugin/pull/313/files#diff-830a3324f5d10ec569eaf1c45a46c3859d022d2980c3ce2dd538ff4c5555bd14R20

Reply:
You can also check in PR I pasted how to suppress phpstan errors that will popup. See `phpstan.neon` file.